examples/webgl_geometry_cube.html

```diff
@@ -33,6 +33,8 @@
 
 			function init() {
 
+				console.log( JSON.stringify( new THREE.Quaternion() ) );
```

Comment:
@Mugen87 Is it a debug statement?

Reply:
Thanks for noticing! e65edb973f1faa6ea68cdc2596b2ffb47619adc8